linux/amzn/2014.09/foxpass_setup.py

```diff
@@ -105,14 +114,15 @@ def run_authconfig(uri, base_dn):
     os.system(cmd)
 
 
-def configure_sssd(bind_dn, bind_pw):
+def configure_sssd(bind_dn, bind_pw, backup_ldaps):
     from SSSDConfig import SSSDConfig
 
     sssdconfig = SSSDConfig()
     sssdconfig.import_config('/etc/sssd/sssd.conf')
 
     domain = sssdconfig.get_domain('default')
     domain.add_provider('ldap', 'id')
+    domain.set_option('ldap_backup_uri', ','.join(backup_ldaps))
```

Comment:
I tested both with and without a proxy configured.  I did not see any errors, but did not specifically look for that line to be causing errors.  I didn't even see any warnings.